ctaes-cbc.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _CTAES_CBC_H_
+#define _CTAES_CBC_H_ 1
+
+#include "ctaes.h"
+
+typedef struct {
+    AES128_ctx ctx;
+    uint8_t iv[16];
```

Comment:
Perhaps document that this isn't exactly the IV (after the first block, it's the carried-forward previous ciphertext).